feature_server/map.py

```diff
@@ -92,9 +89,9 @@ def apply_script(self, protocol, connection, config):
             protocol, connection = self.script(protocol, connection, config)
         return protocol, connection
 
-    def load_vxl(self, rot_info, load_dir):
+    def load_vxl(self, rot_info):
         try:
-            fp = open(rot_info.get_map_filename(load_dir), 'rb')
+            fp = open(rot_info.get_map_filename(self.load_dir), 'rb')
```

Comment:
`self.load_dir` is not set to an instance anyware.. Were those 2 line changes unnecessary? Rolling back those 2 lines fixed map loading.